app/helpers/avo/application_helper.rb

```diff
@@ -67,48 +67,18 @@ def button_classes(extra_classes = nil, color: nil, variant: nil, size: :md, act
     def svg(file_name, **args)
       return if file_name.nil?
 
-      options = {}
-      options[:class] = args[:class].present? ? args[:class] : ""
-      options[:class] += args[:extra_class].present? ? " #{args[:extra_class]}" : ""
+      file_name = "#{file_name}.svg" unless file_name.end_with? ".svg"
 
-      if args[:'data-target'].present?
-        options[:'data-target'] = args[:'data-target']
-      end
-      if args[:'data-tippy'].present?
-        options[:'data-tippy'] = args[:'data-tippy']
-      end
-      if args[:title].present?
-        options[:title] = args[:title]
-      end
-
-      # Create the path to the svgs directory
-      file_path = "#{Avo::Engine.root}/app/assets/svgs/#{file_name}"
-      file_path = "#{file_path}.svg" unless file_path.end_with? ".svg"
-
-      # Create a cache hash
-      hash = Digest::MD5.hexdigest "#{file_path.underscore}_#{options}"
-
-      svg_content = Avo::App.cache_store.fetch "svg_file_#{hash}", expires_in: 1.week, cache_nils: false do
-        if File.exist?(file_path)
-          file = File.read(file_path)
+      paths = [
+        Rails.root.join(file_name).to_s,
```

Comment:
```suggestion
       Rails.root.join('app', 'assets', 'svgs', file_name).to_s,
```
Can we have this, to avoid putting full path, the default folder for svgs can be `svgs` in app/assets

Reply:
This will also enable as to retain how icons we created in the previous version
```
  section I18n.t('avo.dashboard'), icon: 'dashboard', collapsable: true, collapsed: true do

 end
```

Reply:
Yes. Backward compatibility is something that I want to keep as well. 
Thanks for you input!

Reply:
Does [this](https://github.com/avo-hq/avo/pull/1141/files#diff-27ec89c6677b158fe3a800b268c9c196b5409f439315240504626f4c8bf81bc7R78-R81) look good enough for you @thirunjuguna?